agent/nmap_agent.py

```diff
@@ -394,6 +415,50 @@ def _emit_fingerprints(
                                 data=msg_data,
                             )
 
+    def _connect_to_vpn(self) -> None:
+        """Connect to VPN."""
+        logger.info("Trying to scan the asset with VPN")
+        try:
+            with open(WIREGUARD_CONFIG_FILE_PATH, "w", encoding="UTF-8") as conf_file:
+                conf_file.write(cast(str, self._vpn_config))
+            with open(DNS_RESOLV_CONFIG_PATH, "w", encoding="UTF-8") as conf_file:
+                conf_file.write(cast(str, self._vpn_config))
+
+            self._exec_command(
```

Comment:
why copy the content of the file to another file since the place of the file is known you can use it directly

Reply:
+1 here.